guardian/itunes-rss
# Use fastly-resizer image URLs for specific podcast

## What does this change?

For the ~AU full-story~ Grace Dent Comfort Eating podcast we want to use episodic artwork if possible. This tests the water in that area.

## How to test

Locally you can export the necessary env vars (see the README) and run it. It'll generate the <itunes:image> elements for appropriate podcast's episodes only. The links won't actually work unless you export the correct salt. The salt also needs to be added to our configuration for access by `SecretKeeper`.

## How can we measure success?

If we see per-episode images... success!

## Have we considered potential risks?

* Currently there are no coded protections against older series episodes - we may need a date-based gate to enable this in production.
* We exclude master assets from being considered for selection - this may or may not have to remain
* We don't specify an upper- or lower-bound size limit on the assets we do select - this may have to be added
* Regardless of which image we select we'll always ask Fastly to make a 3000px square crop - this may have to be adjusted according to the size of the image asset we select

## Images

TBA

## Accessibility
